gatsby-node.js

```diff
@@ -11,7 +11,7 @@ const { createFilePath } = require(`gatsby-source-filesystem`)
 exports.onCreateNode = ({ node, getNode, actions }) => {
   const { createNodeField } = actions
   if (node.internal.type === `PeopleYaml`) {
-    const slug = `/people/${node.name}-${node.lastname.replace(/ /g, '-')}`
```

Comment:
This should not be reverted. Related to #147.

Reply:
thanks for reviewing, have no idea why this line was edited, maybe I accidentally touched it.